tools/to_ipynb.py

```diff
@@ -0,0 +1,116 @@
+import os
```

Comment:
I don't want to enforce any practice or similar, but I normally use Cider which formats code quite nicely Using PEP-8 standard.
I add this file to this cl/216603659

Reply:
feel free to point out issues.